src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
+            path = ProjectManager.getProjectRoot().fullPath;
+        }
+
+        return path;
+    }
+
     function _loadBowerJson(path) {
         _bowerJsonFile = new BowerJsonFile(path);
     }
 
+    function getDependencies() {
+        var deferred = new $.Deferred();
+
+        _bowerJsonFile.getContent()
+            .done(function (data) {
+                var dependencies = [];
+                data = JSON.parse(data);
+
+                _.each( data.dependencies, function (version, name) {
+                    dependencies.push({name:name, version:version});
+                } );
+
```

Comment:
You're right. I can live with this now. But that wasn't my intention when I modeled the BowerJsonFile and ConfigurationFile (I think this is the name...). BowerFile only handle the basis, create delete etc through FileUtils.
BowerJsonFile should handle dependencies and devDependencies, since for now, these are the most important properties. 
We can discuss this in another thread and with more time of my own :) I'm in a hurry now.